kubelet-to-gcm/monitor/kubelet_to_gcm.go

```diff
@@ -0,0 +1,412 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package monitor
+
+import (
+	"fmt"
+	"time"
+
+	v3 "google.golang.org/api/monitoring/v3"
+	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/stats"
+)
+
+var (
+	reservedCoresMD = &metricMetadata{
```

Comment:
These are the GCM/stack driver metrics. I'm pushing master component metrics to stack driver from GKE's hosted masters.